multi-repo-action/org-workflow-add.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/google/go-github/v42/github"
+	"golang.org/x/oauth2"
+)
+
+// *****SET THESE PARAMETERS*****
+const ORG_NAME string = "organization name"
```

Comment:
orgs/individual may want to install only on a subset of the repos. This is something that was requested, so let's add support for it.

Reply:
How do we define the subset? Is it by a certain property, or should we allow them to provide a list of repos?

Reply:
I suppose a list of repo, or a config file. feel free to suggest.

Reply:
Added repo list as a parameter